docs/installation/imgproxy.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Imgproxy
+---
+
+Imgproxy is a standalone server for resizing images. It is optional and you do
+not need it to start Forem. We are currently only it using Forem-cloud's
+application.
+
+## Installation
+
+On macOS, use homebrew with `brew install imgproxy`. For others, please see the
+[official documentation](https://docs.imgproxy.net/#/installation)
+
+## Usage
+
+1. To enable Imgproxy in development, first add
+   `export IMGPROXY_ENDPOINT="http://localhost:8080"` to your `.env`
+1. Startup Imgproxy in a terminal. If you installed via homebrew, it's
+   `imgproxy`.
+1. Startup the Forem app.
+1. That's it :)
+
+You should verify it's working by checking a user's profile image. It should
+have a structure similar to
+`http://localhost:8080/unsafe/rs:fill:320:320/aHR0cDovL2xvY2Fs/aG9zdD.......`.
+You should also see activities from Imgproxy terminal and all the images should
+load.
```

Comment:
Is there a `curl` command we can use to check? Like fetching one of the builtin images

Reply:
I just added a curl command but just realized I'm making the assuming that our users would use `localhost:3000`. Since the url is encoded it could potentiall be confusing. What do you think?

Reply:
maybe we can replace that with https://docs.imgproxy.net/#/healthcheck?id=imgproxy-health but it might be useless as we're testing the server, not the proxy to our app :D Nevermind

Reply:
Haha, well I think I have there is better than what I had before so I'm going to leave it as is for the time being.